.buildkite/pipeline.verify.ami-test.yml

```diff
@@ -0,0 +1,26 @@
+# This file borrows heavily from the pattern defined in `pipeline-infrastructure`.
+
+---
+env:
+  PANTS_CONFIG_FILES: "['pants.toml', 'pants.ci.toml']"
+
+# This is defined in a separate pipeline file because
+# chronotc/monorepo-diff does not currently have support for plugin
+# (or key) configuration on commands.
+#
+# :(
+
+steps:
+  # TODO: lint packer
```

Comment:
Linting can take place in the main `verify` pipeline; it should only take a few seconds, so there's no reason to make it conditional.

Reply:
done